src/app/feedback-form/feedback-form.component.html

```diff
@@ -18,9 +21,22 @@ <h3 class="contact-form__title">Get in Touch with Us!</h3>
           class="contact-form__input"
           type="email"
           id="email"
-          name="email"
-          required
+          formControlName="email"
         />
+        <div
+          class="error-message"
+          [hidden]="
```

Comment:
hidden makes the element still take up some space even if its not displayed. Are oyu sure you want that?